ssg/checks.py

```diff
@@ -54,7 +54,33 @@ def is_cce_valid(cceid):
     http://people.redhat.com/swells/nist-scap-validation/scap-val-requirements-1.2.html
     """
     match = re.match(r'^CCE-\d{4,5}-\d$', cceid)
-    return match is not None
+    if match is None:
```

Comment:
@cipherboy should we have this check in a separate function so that we can have better error messages as to why the cce check failed?

Reply:
The error message is raised at a higher level than this function, e.g., [here](https://github.com/openscap/scap-security-guide/blob/master/ssg/build_yaml.py#L555). We'd have to turn it into two function calls and update it in every location (`is_cce_valid_format` + `is_cce_valid_contents` or some such), with different error messages for each. 

It is only used in two places, one of which errors (`build_yaml.py` line 555), the other just prints a warning (`build_renumber.py` line 428).

Reply:
I think we should update the error messages; otherwise, it is going to be cryptic as to why it failed. Another idea is maybe return the errors or raise the errors in the function itself?